Tmain/client-vista-vim-fields-expectation.d/run.sh

```diff
@@ -0,0 +1,11 @@
+# Copyright: 2021 <liuchengxu@github.com>
+# License: GPL-2
+#!/usr/bin/env bash
+
+CTAGS=$1
+
+. ../utils.sh
+
```

Comment:
Could you add
```
is_feature_available "${CTAGS}" json
```
This line is for skipping this test case if ctags has no json writer.